bigquery_etl/events_daily/query_templates/event_types/templating.yaml

```diff
@@ -0,0 +1,4 @@
+org_mozilla_firefox:
+  app_id: org_mozilla_firefox
+telemetry:
+  app_id: telemetry
```

Comment:
Can you add a comment here about why we set `app_id: telemetry`? This feels like a made-up value to satisfy some constraint, so needs explanation.